include/envoy/singleton/instance.h

```diff
@@ -13,5 +13,19 @@ class Instance {
 
 typedef std::shared_ptr<Instance> InstanceSharedPtr;
 
+/**
```

Comment:
nit: IMO this should go in manager.h, but I don't feel that strongly about it if you prefer it here.